virt/virtual_machines/virt-create-vms.adoc

```diff
@@ -30,12 +30,14 @@ Due to differences in storage behavior, some virtual machine templates are incom
 
 include::modules/virt-creating-vm-quick-start-web.adoc[leveloffset=+1]
 
-include::modules/virt-creating-vm-wizard-web.adoc[leveloffset=+1]
+include::modules/virt-quick-creating-vm.adoc[leveloffset=+1]
 
-Refer to the virtual machine wizard fields section when running the web console wizard.
+include::modules/virt-creating-vm-custom-template.adoc[leveloffset=+1]
+
+Refer to the virtual machine fields section when creating a VM from the web console.
 
 :virtualmachine:
-include::modules/virt-vm-wizard-fields-web.adoc[leveloffset=+2]
+include::modules/virt-vm-fields-web.adoc[leveloffset=+2]
 
 Enable the xref:../../scalability_and_performance/using-cpu-manager.adoc#using-cpu-manager[CPU Manager] to use the high-performance workload profile.
```

Comment:
I know that you did not add this link. I just noticed it. It looks a bit lost here. I think it would be better to put the link under Additional Resources.

A note about CPU Manager would be useful close to the workload profile setting, but I do not see "profile" anywhere in our settings. I would have to check one of the video demos to be certain.

Reply:
You might want to make the following modules (networking, storage, cloud-init) leveloffset=+3 to show that they are subordinate to the VM fields. Otherwise, the user might wonder how they fit into the VM network fields.